pysrc/qulacs/converter/qasm_converter.py

```diff
@@ -12,6 +12,10 @@
 logger = getLogger(__name__)
 logger.addHandler(NullHandler())
 
+FIXED_POINT_PATTERN = r"[+-]?\d+(?:\.\d*)?|\.\d+"
+FLOATING_POINT_PATTERN = r"[+-]?[eE][-+]?\d+"
```

Comment:
```suggestion
FLOATING_POINT_PATTERN = r"[eE][-+]?\d+"
```
With this `[+-]?`, `1+e1` matches and it's incorrect.

Reply:
Good suggestion, thank you!